### ISSUE TITLE
Panoptic net nan loss during training

### ISSUE THREAD

**Issue by msschwartz21** [vanvalenlab/deepcell-tf#515]
**Describe the bug**
This issue was discovered while training a cell segmentation model for the class. The corrected example is available [here](https://vanvalenlab.github.io/bebi205/notebooks/segmentation.html)

**Dave's Notes**
The nan loss stems from not having a softmax layer at the end for that semantic head instead its an activation layer with a relu
you can see it in the model if you use plot_model. Its not really a bug per se, but the default setting for the library is that include_top for create_semantic_head is false which means you never add the softmax layer and it defaults to activation. What it should do is check the channel dimension - if it is 1 use activation if > 1 use softmax and it will do that if include_top is set to true.

**Comment by willgraf** [comment]
Does this mean we should remove `include_top` from `create_semantic_head` entirely and just use the channel dimension to decide?

**Comment by msschwartz21** [comment]
I first found the error with the following code:
```python
from deepcell.model_zoo.panopticnet import PanopticNet

semantic_classes = [1, 1, 2] # inner distance, outer distance, fgbg

model = PanopticNet(
    backbone='resnet50',
    input_shape=X_train.shape[1:],
    norm_method='whole_image',
    num_semantic_classes=semantic_classes)
```
When I tried to train the model, the losses were infinite:
```
Epoch 1/5
204/204 [==============================] - 89s 369ms/step - loss: nan - semantic_0_loss: 0.0805 - semantic_1_loss: 0.0224 - semantic_2_loss: nan - val_loss: nan - val_semantic_0_loss: 0.0017 - val_semantic_1_loss: 0.0067 - val_semantic_2_loss: nan
```
The problem was fixed by adding `include_top=True`. 

Dave originally diagnosed the issue so he may have better insight on what the correct default behavior should be. I think the issue would be resolved by defaulting `include_top` to `True`, but I'm not sure if that would be a problem in other cases.
